framework/src/main/java/org/checkerframework/framework/flow/CFAbstractValue.java

```diff
@@ -446,7 +446,7 @@ private final AnnotationMirror lowestQualifier(
      * instead of this method.
      */
     @Override
-    public final V leastUpperBound(@Nullable V other) {
+    public V leastUpperBound(@Nullable V other) {
```

Comment:
Can't CFI override the method mentioned in the javadoc instead?

Reply:
Yes, I think it can. I just override the commits.